docs/dev/reference/sdks/connectivity.md

```diff
@@ -15,6 +15,76 @@ date: "2022-01-01"
 
 When connecting to a machine using the connection code from the [**CONNECT** tab](/dev/reference/sdks/), a [client session](/dev/reference/apis/sessions/) automatically uses the most efficient route to connect to your machine either through local LAN or WAN or the internet.
 
+## Connect over local network
+
+To connect directly to the local machine, change the end of the machine address URI from `.viam.cloud` to `.local.viam.cloud` in your machine's connection code.
+The default machine address (URI) from the **CONNECT** tab is of the form `mymachine-main.0a1bcdefgi.viam.cloud`, so `mymachine-main.0a1bcdefgi.local.viam.cloud`.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python {class="line-numbers linkable-line-numbers" data-line="10"}
+async def connect():
+    opts = RobotClient.Options.with_api_key(
+        # Replace "<API-KEY>" (including brackets) with your machine's API key
+        api_key='<API-KEY>',
+        # Replace "<API-KEY-ID>" (including brackets) with your machine's
+        # API key id
```

Comment:
```suggestion
        # API key ID
```
[nit/opt] Same for all api -> API as well but this is everywhere so can disregard